dragonfly/block/register.go

```diff
@@ -133,6 +133,12 @@ func init() {
 	item_internal.Air = Air{}
 	item_internal.Grass = Grass{}
 	item_internal.GrassPath = Grass{Path: true}
+	item_internal.FarmLand = Farmland{Hydration: 7}
+	item_internal.IsFarmland = func(b world.Block) bool {
```

Comment:
This doesn't seem to be used anywhere.

Reply:
FarmLand is used with hoes, and when it comes time to implement naturally generated structures with crops this will become handy

Reply:
These functions like IsFarmland are only there to work around circular dependency. If they're not actually needed (yet), there's no need to have them yet.

Reply:
Okay, I can remove them